distribution/docs/src/main/resources/content/_using/using-catalog-search-ui.adoc

```diff
@@ -263,7 +263,7 @@ Bookmarked results are denoted with a yellow *star* (image:bookmarked-icon.png[]
 
 Visuals are different ways to view search results.
 
-. Click the *Add Visual* (image:add-visual-icon.png[add visual icon]) icon in the bottom right corner of ${catalog-ui}.
+. Click the *[ + ]* icon in the top right corner of ${catalog-ui}.
```

Comment:
Is this still true?

Reply:
kind of - went ahead and added the image of the new button instead of the textual representation